src/sentry/static/sentry/app/views/settings/organizationIntegrations/configureIntegration.tsx

```diff
@@ -206,7 +206,7 @@ class ConfigureIntegration extends AsyncView<Props, State> {
   renderMainContent(provider: IntegrationProvider) {
     const {integration} = this.state;
     //if no code mappings, render the single tab
-    if (!this.hasStacktraceLinking(provider)) {
+    if (this.hasStacktraceLinking(provider)) {
```

Comment:
whoops

Reply:
@EvanPurkhiser yea this broke everything hahaha